crates/matrix-sdk-crypto/src/backups/mod.rs

```diff
@@ -555,6 +555,58 @@ impl BackupMachine {
 
         (backup, session_record)
     }
+
+    /// Import the given room keys into our store.
+    ///
+    /// # Arguments
+    ///
+    /// * `room_keys` - A list of previously exported keys that should be
+    /// imported into our store. If we already have a better version of a key
+    /// the key will *not* be imported.
+    ///
+    /// Returns a [`RoomKeyImportResult`] containing information about room keys
+    /// which were imported.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # use std::io::Cursor;
+    /// # use matrix_sdk_crypto::{OlmMachine, decrypt_room_key_export};
+    /// # use ruma::{device_id, user_id};
+    /// # let alice = user_id!("@alice:example.org");
+    /// # async {
+    /// # let machine = OlmMachine::new(&alice, device_id!("DEVICEID")).await;
+    /// # let export = Cursor::new("".to_owned());
+    /// let exported_keys = decrypt_room_key_export(export, "1234")?;
+    /// machine.backup_machine().import_backed_up_room_keys(exported_keys, |_, _| {}).await?;
+    /// # anyhow::Ok(())
+    /// # }
+    /// ```
+    pub async fn import_backed_up_room_keys(
+        &self,
+        room_keys: BTreeMap<OwnedRoomId, BTreeMap<String, BackedUpRoomKey>>,
```

Comment:
Could we use a type alias for the `String` part here (or even better, a transparent newtype wrapper), to document a bit better what it is? (It's a session id from looking at the code below, but this won't be obvious to someone reading docs.rs)

Reply:
I guess I can add a type alias, but we should perhaps consider adding a newtype wrapper to Ruma, since the type here is kept the same as the response: https://docs.rs/ruma/latest/ruma/api/client/backup/struct.RoomKeyBackup.html#structfield.sessions. Ok, minus the fact that we pass the decrypted room key here.